R/dash.R

```diff
@@ -374,55 +344,24 @@ Dash <- R6::R6Class(
     # ------------------------------------------------------------------------
     # callback registration
     # ------------------------------------------------------------------------
-    callback = function(func = NULL, output = NULL) {
+    callback = function(output = NULL, inputs = list(), state = list(), user_function) {
```

Comment:
I think we should rename `user_function` into something more human-readable / conventional. The natural name would be, well, `callback`, but that feels redundant. Maybe `callback_func` or just `func`?

Reply:
Agreed, I think `func` would be preferable instead.